src/Microsoft.VisualStudio.VsInteractiveWindow/VsInteractiveWindow.cs

```diff
@@ -122,6 +127,13 @@ private void SubmissionBufferAdded(object sender, SubmissionBufferAddedEventArgs
             GetToolbarHost().ForceUpdateUI();
         }
 
+        private void OnFramePropertyChanged(object sender, PropertyChangedEventArgs eventArgs)
+        {
+            if (eventArgs.PropertyName.Equals("Title", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
Roslyn has most of the relevant ones defined here: https://github.com/dotnet/roslyn/blob/master/.editorconfig as does CoreFX https://github.com/dotnet/corefx/blob/master/.editorconfig (CoreFX also defines some of the naming rules we have).